components/browser/state/src/test/java/mozilla/components/browser/state/state/TabPartitionTest.kt

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.state.state
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class TabPartitionTest {
+
+    @Test
+    fun `GIVEN a null tab partition THEN tab partition is empty`() {
+        val tabPartition: TabPartition? = null
+
+        assertEquals(tabPartition.isEmpty(), true)
```

Comment:
nit: `isEmpty` and `isNotEmpty` return `Boolean` so should be able to use `assertTrue`/`assertFalse` for all of these.